ingestion/Dockerfile

```diff
@@ -29,7 +29,19 @@ RUN apt-get update \
     wget --no-install-recommends \
     # Accept MSSQL ODBC License
     && ACCEPT_EULA=Y apt-get install -y msodbcsql18 \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
+
+# Oracle instant client for thick mode
+# https://python-oracledb.readthedocs.io/en/latest/user_guide/initialization.html#enablingthick
+RUN apt-get update; \
+  apt-get install libaio1 alien unzip -y;
```

Comment:
Instead of separately running another Docker `RUN` Instruction, let's include these in the already available `RUN` instruction which installs apt dependencies!
![image](https://user-images.githubusercontent.com/15995028/213214005-6a98016e-3b1a-4cc1-8c47-1d05de2073c3.png)